quarkus-build-caching-extension/.mvn/develocity.xml

```diff
@@ -1,19 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
 
-<gradleEnterprise
-        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+<develocity
+        xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
     <server>
         <url>https://ge.solutions-team.gradle.com</url>
         <allowUntrusted>false</allowUntrusted>
```

Comment:
`false` is already the default value so it could be removed. I think it has always been this way but now would be a good time to change it.

Reply:
done, absolutely 💯